pyteal/ast/ternaryexpr.py

```diff
@@ -77,6 +77,24 @@ def Ed25519Verify(data: Expr, sig: Expr, key: Expr) -> TernaryExpr:
     )
 
 
+def Ed25519Verify_Bare(data: Expr, sig: Expr, key: Expr) -> TernaryExpr:
+    """Verify the ed25519 signature of the data against the public key.
+
+    Args:
+        data: The data signed by the public key. Must evalutes to bytes.
```

Comment:
@jdtzmn Minor:  I spotted a few typos and opened https://github.com/jdtzmn/pyteal/pull/2.  Can you review + merge if it looks good to you?